runtime/storage/sqldb/stdlib_wrapper_internal.go

```diff
@@ -321,6 +323,17 @@ var (
 	_ driver.NamedValueChecker  = wrappedConn{}
 )
 
+// Conn exposes the underlying pgx database connection so that it can be
+// accessed via (*sql.Conn).Raw.
+//
+//	conn.Raw(func(driverConn any) error {
+//		pgxConn := driverConn.(interface{ Conn() *pgx.Conn }).Conn()
+//		// do stuff with *pgx.Conn
+//	})
+func (c wrappedConn) Conn() *pgx.Conn {
+	return (c.parent).(*stdlibdriver.Conn).Conn()
+}
```

Comment:
This is the second way which is a bit more direct, piggy-backing on `(*sql.Conn).Raw`. I took some inspiration from the implementation of [`errors.Unwrap`](https://cs.opensource.google/go/go/+/refs/tags/go1.20.2:src/errors/wrap.go;l=16-24) in the example code which type asserts that `driverConn` is actually wrapping `*pgx.Conn` and exposes it via `Conn()`.

I suppose the benefit of this approach is that client code could type assert `driverConn` in a switch statement without having to import any packages from Encore, so hypothetically a client package could support a `*sql.Conn` that _doesn't_ have another layer of wrapping over `*pgx.Conn` like Encore's.

I suspect though that the compile-time type checking of the `DriverConn` approach is superior in practice.